apps/server/src/stores/runtimeState.ts

```diff
@@ -136,10 +136,12 @@ export function clear() {
  * Utility to allow modifying the state from the outside
  * @param newState
  */
-function patchTimer(newState: Partial<TimerState>) {
+function patchTimer(newState: Partial<TimerState & RestorePoint>) {
   for (const key in newState) {
     if (key in runtimeState.timer) {
       runtimeState.timer[key] = newState[key];
+    } else if (key in runtimeState._timer) {
+      runtimeState._timer[key] = newState[key];
```

Comment:
the _timer tries to signify that this is a private property that we use to save some state. I dont think we should be passing it around or expose it externally

Reply:
I is strictly needed for the restore service to work properly
currently it restores to a strange state. If you pause a playback and force restart the server it will look like pause but continue playing

Reply:
Nah, sorry. Just leave it as you have it. Would be nice to enable strict mode in the server eventually

Reply:
Just add that explanation to the code